### PR TITLE
Adding NFT to the emscripten version and improving compilation ( libjpeg )

### PR DESCRIPTION
**Pull request by kalwalt**
This PR add NFT support to the emscripten branch and simplify and correct the libjpeg includes and libs for building the project.
By emscripten **v1.38.31** is only necessary to add the flags `-s USE_LIBJPEG=1` and emscripten will add the includes and the libjpeg lib. see https://github.com/emscripten-core/emscripten/issues/9170 and recent changes in jsartoolkit5 https://github.com/kalwalt/jsartoolkit5/pull/13
This is necessary if we want add NFT see my PR https://github.com/augmentmy-world/artoolkitX.js/pull/18

### CODE REVIEW

File: Source/artoolkitx.js/CMakeLists.txt [hunk 1]
```diff
@@ -24,7 +24,7 @@ set(MODULE_OPTIONS "-s MODULARIZE_INSTANCE=1 -s EXPORT_NAME='artoolkitXjs' -s EX
 
 math(EXPR MEM 256*1024*1024)
 message("CMAKE_BUILD_TYPE: ${CMAKE_BUILD_TYPE}")
-set(MEMORY "-s TOTAL_MEMORY=${MEM}")
+set(MEMORY "-s TOTAL_MEMORY=${MEM}  -s ALLOW_MEMORY_GROWTH=1")
```
Comment (kalwalt) [inline Source/artoolkitx.js/CMakeLists.txt]: i need to check if this is still necessary

**Comment by kalwalt** [comment]
@ThorstenBux I would like to merge this PR if you have nothing against it. I need to check some little deatils and after i will fine. I don't want to make big PR but i prefer add features step by step.

**Comment by ThorstenBux** [comment]
@kalwalt awesome. Yes small PRs is very good. You may merge if you like :)

**Comment by kalwalt** [comment]
> @kalwalt awesome. Yes small PRs is very good. You may merge if you like :)

Yes, i need to check only if the `-s ALLOW_MEMORY_GROWTH=1` is needed or not for the NFT, maybe needed for libjpeg?

**Comment by ThorstenBux** [comment]
It was needed before I used libJPEG from Emscripten. It should not be needed anymore.

**Comment by kalwalt** [comment]
Yes it's not needed! I'm merging this. :slightly_smiling_face:

**Pull request merged**